ros_ws/src/interpreter_pkg/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>interpreter_pkg</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="vagrant@todo.todo">vagrant</maintainer>
+  <license>TODO: License declaration</license>
+  
+  <depend>os</depend>
+  <depend>rclpy</depend>
+  <depend>std_msgs</depend>
+  <depend>langchain</depend>
```

Comment:
You should also be dependent on capstone_interfaces for your custom message, no?

Reply:
I didnt do a custom message.  I just used stock String